### PR TITLE
Perf/sheet

### PR DESCRIPTION
**Pull request by hkjels**

### CODE REVIEW

**Comment by hkjels** [comment]
Huge PR. I know, SRY

File: src/cljc/ui/wire/polyglot.cljc [hunk 1]
```diff
@@ -31,6 +31,8 @@
    :ui/month             (tongue/inst-formatter "{month-long}" inst-strings-en)
    :ui/weekday-long      (tongue/inst-formatter "{weekday-long}" inst-strings-en)
    :ui/weekday-short     (tongue/inst-formatter "{weekday-short}" inst-strings-en)
+   :sort-ascending       "Sort Ascending"
+   :sort-descending      "Sort Descending"
```
Comment (peilertsen) [inline src/cljc/ui/wire/polyglot.cljc]: Does this mean that there are no translations on this?

Comment (hkjels) [reply src/cljc/ui/wire/polyglot.cljc]: No, we just need some translations in `ui` for demo-purposes

**Review by kleinrein — APPROVED** [review]
Looks good 👍

**Pull request merged**